python/src/deltachat/_build.py

```diff
@@ -0,0 +1,23 @@
+import subprocess
+import tempfile
+
+import cffi
+
+
+ffibuilder = cffi.FFI()
+ffibuilder.set_source(
+    'deltachat.capi',
+    """
+    #include <deltachat/mrmailbox.h>
+    """,
+    libraries=['deltachat'],
+)
+with tempfile.NamedTemporaryFile(mode='r') as fp:
+    proc = subprocess.run(['gcc', '-E', '-o', fp.name, '-DPY_CFFI=1',
+                           '../src/mrmailbox.h'])
```

Comment:
Clean up this implementation, e.g. don't hardcode gcc, don't hardcode path to header file.